common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -872,18 +872,27 @@ def assert_page(note, usage_id, view):
 
         self._add_default_notes()
         self.notes_page.visit()
+        # visiting the page results in an ajax request to fetch the notes
+        self.notes_page.wait_for_ajax()
```

Comment:
@jzoldak I had thought about it, but because of the .visit calls,it would have to go into is_browser_on_page. Seems odd to have a wait statement there, though I could be convinced. (That's how I had it before submitting the PR. It was a one-line change.)